packages/integrations/package.json

```diff
@@ -35,7 +35,6 @@
     "localforage": "^1.8.1"
   },
   "devDependencies": {
-    "@sentry/browser": "7.111.0",
```

Comment:
This lead to a circular dependency, and we only used it in a single test for `WINDOW` anyhow 😬